src/plugins/dragresize_ie.js

```diff
@@ -878,10 +878,9 @@
                 // Don't update attributes if less than 10.
                 // This is to prevent images to visually disappear.
                 if (newWidth >= 15 && (newHeight >= 15 || newHeight === 'auto')) {
-                    image.setAttributes({
-                        width: newWidth,
-                        height: newHeight
-                    });
+                    image.$.style.width = newWidth;
```

Comment:
Since we're using styles, shouldn't the value should be in pixels?

Reply:
You're absolutely right @julien I had used pixels in my testing version of the fix and must've forgotten it when I transferred it over to send the pull. I'll resend it with that change on a new PR.